cost/superseded_instance/superseded_instance.pt

```diff
@@ -410,20 +416,38 @@ end
 policy "pol_aws_superseded_instance" do
   validate_each $ds_combined_instance_data do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} possible superseded instances"
-    detail_template <<-EOS
-# Superseded Instances
-
-| Resource UID | Name | Current Instance Type | Region | New Instance Type | VPC Required | ENA Required | Monthly Estimated Cost |
-| ------------ | ---- | --------------------- | ------ | ----------------- | ------------ | ------------ | ---------------------- |
-{{ range data -}}
-| {{ if .warn }}**_{{ end -}}{{ .resource_id }}{{ if .warn }}_**{{ end -}} | {{ .name }} | {{.instance_type}} | {{.region}} | {{.new_instance_type}} | {{ if .vpc_incompatible_move }} **_TRUE_** {{else}} false {{end}} |  {{ if .ena_incompatible }} **_TRUE_** {{else}} false {{end}} | {{.run_rate}} |
-{{ end -}}
-
-___
-###### Policy Applied in Account: {{ rs_project_name }} (Account ID: {{ rs_project_id }}) within Org: {{ rs_org_name }} (Org ID: {{ rs_org_id }})
-EOS
     check logic_or(lt(to_n(val(item,"run_rate")),$param_threshold), lt(to_n(val(item,"run_rate")),$param_minimum_savings_threshold))
     escalate $report_instances
+    export do
+      resource_level true
+      field "id" do
+        label "Instance ID"
+      end
+      field "resource_id" do
```

Comment:
I don't know why you added this, isn't the same as Id?

Reply:
I was not sure if they are the same.  My intention was to keep everything in the summary section https://github.com/flexera/policy_templates/blob/master/cost/superseded_instance/superseded_instance.pt#L416. Would you like me to remove the Resource UID field?